tools/python3/cmake.py

```
@@ -0,0 +1,48 @@
+    return [line.split(" ")[-1] for line in out.splitlines()[1:-2]]
+
+
+def test_cmd(test, verbose=False):
+    cmd = f"ctest -R {test}"
+    if verbose:
+        cmd = f"{cmd} -V"
+    return cmd
+
+
+def run_test(test, verbose=False, capture_output=False):
+    run(test_cmd(cmd, verbose=verbose), capture_output=capture_output)
```

Ruff (0.8.2)

Undefined name `cmd`

(F821)